lib/posix/options/Kconfig.device_io

```diff
@@ -42,6 +42,14 @@ config POSIX_DEVICE_IO_ALIAS_WRITE
 	help
 	  Select 'y' here and Zephyr will provide an alias for write() as _write().
 
+config POSIX_DEVICE_IO_STDIN_STDOUT_STDERR
+	bool
+	help
+	  Select 'y' here and Zephyr will provide the stdin, stdout, and stderr variables.
```

Comment:
The help text is slightly misleading as user is not able to y this option because it is hidden.

Reply:
Will fix after the TSC meeting tomorrow.

Reply:
Should be fixed now. I applied the same pattern to other non-user-selectable Kconfig as well.